Sources/AirRobeWidget/Resources/AirRobeStrings.swift

```diff
@@ -9,33 +9,33 @@ import Foundation
 
 enum AirRobeStrings {
     // MARK: - Title Strings
-
     static let added = "Added To"
     static let add = "Add To"
 
     // MARK: - Description Strings
-
     static let description = "Wear now, re-sell later."
     static let descriptionCutOffText = "Re-sell later."
 
     // MARK: - Potential Value Strings
-
     static let potentialValue = "Potential value: "
 
     // MARK: - Detailed Description Strings
-
     static let detailedDescription = "We’ve partnered with AirRobe to enable you to join the circular fashion movement. Re-sell or rent your purchases on AirRobe’s marketplace – all with one simple click. Together, we can do our part to keep fashion out of landfill. Learn more."
     static let learnMoreLinkText = "Learn more."
+    static let learnMoreLinkForPurpose = URL(string: "https://airrobe.com/")!
 
     // MARK: - Extra Info Strings
-
-    static let extraInfo = "By opting in you agree to THE ICONIC’s Privacy Policy and consent for us to share your details with AirRobe."
+    static let extraInfo = "By opting in you agree to COMPANY NAME’s Privacy Policy and consent for us to share your details with AirRobe."
```

Comment:
Should probably use the variable `companyNameText` here instead of having a string literal on line 28 and on line 30. It's a bug waiting to happen in the future.

Safer: 

```swift
    static let companyNameText = "COMPANY NAME"
    static let extraInfo = "By opting in you agree to \(companyNameText)’s Privacy Policy and consent for us to share your details with AirRobe."
    static let extraLinkText = "Privacy Policy"

```

Reply:
Good point 👍 

> > > @creative-dev-lab Random thought, but adding the Cart button to the NavigationBar on all screens would cut down on testing time.
> > 
> > 
> > That's a good point! 👍 I will add the cart button to the navigation bar on all screens in the next PR for sure.
> 
> @creative-dev-lab Eli, in general we like to resolve issues in the PR they were reported. Could you make the changes in this PR rather than the next one? Just so we have the full context of the discussion here along with the changes. There are no urgent updates for the mobile SDKs, so I think we can keep this PR open and resolve Eric's recommendations here. It would be much easier for the team to follow this way.

Yep, no problem with that.